Gallery.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Gallery page</title>
+    <style>
+* {
+  box-sizing: border-box;
+}
+body { 
+      margin: 0;
+      font-family: sans-serif;
+    }
+    
+    .header1 {
+      overflow: hidden;
+      padding: 20px 10px;
+      background-color: black;
+    }
+    
+    .header1 a {
+      float: left;
+      color: white;
+      text-align: center;
+      padding: 12px;
+      text-decoration: none;
+      font-size: 18px; 
+      line-height: 25px;
+      border-radius: 4px;
+    }
+    
+    .header1 a.logo {
+      font-size: 25px;
+      font-weight: bold;
+    }
+    
+    .header1 a:hover {
+      background-color: #ddd;
+      color: black;
+    }
+    
+    .header1 a.active {
+      background-color: dodgerblue;
+      color: white;
+    }
+    
+    .header-right {
+      float: right;
+    }
+    
+    /* @media screen and (max-width: 500px) {
+      .header1 a {
+        float: none;
+        display: block;
+        text-align: left;
+      } */
+      
+      /* .header1-right {
+        float: none;
+      }
+    } */
+.header {
+  text-align: center;
+  padding: 32px;
+}
+
+.row {
+  display: -ms-flexbox; 
+  display: flex;
+  -ms-flex-wrap: wrap; 
+  flex-wrap: wrap;
+  padding: 0 4px;
+}
+
+.column {
+  -ms-flex: 25%; 
+  flex: 25%;
+  max-width: 25%;
+  padding: 0 4px;
+}
+
+.column img {
+  margin-top: 8px;
+  vertical-align: middle;
+  width: 100%;
+}
+
+@media screen and (max-width: 800px) {
+  .column {
+    -ms-flex: 50%;
+    flex: 50%;
+    max-width: 50%;
+  }
+}
+
+@media screen and (max-width: 600px) {
+  .column {
+    -ms-flex: 100%;
+    flex: 100%;
+    max-width: 100%;
+  }
+}
+</style>
+</head>
+<body>
+
+  <div class="header1">
+    <a href="index.html" class="logo">My Site</a>
+    <div class="header-right">
+      <a href="index.html">Home</a>

```

Comment:
Once look into grids in css. And from now on styling should be in a separate file. approving as of now. but need to follow these from now on